client/src/components/MenuWrapper.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Flex } from "@chakra-ui/core";
+import { OptionsMenu, ConfirmationDialog } from ".";
+
+const MenuWrapper = ({ item, itemType, setOpenConfirmationDialog, openConfirmationDialog, confirm, hasEdit, onEdit }) => {
```

Comment:
Just a thought:

Couldn't you use the `onEdit` prop to infer `isEdit`? (so you wouldn't need the `isEdit` prop).

So in `OptionsMenu.js` maybe you could use `onEdit && ...`.

Reply:
Good point! 😮